citest/json_contract/observation_verifier.py

```diff
@@ -51,7 +51,6 @@ def __init__(self, observation):
     self.__valid_obj_set = []  # Cannot be a set because it is unhashable.
     self.__good_results = []
     self.__bad_results = []
-    self.__all_results = []
```

Comment:
Trying to simplify. all was superfluous. I thought it would be convienent to look at as a whole, but I dont recall ever using it. In practice you just want to see what was bad (if expecting everything to be good) and/or what was good (if expecting some but not all to be good)